tests/integration/test_transformers.py

```diff
@@ -255,7 +256,11 @@ def _test_transformer_with_hypertransformer(transformer_class, input_data, steps
         }
 
     hypertransformer.detect_initial_config(input_data)
-    hypertransformer.update_transformers(field_transformers)
+    try:
+        hypertransformer.update_transformers(field_transformers)
+    except InvalidSdtypeForTransformerError:
+        pass
```

Comment:
I think instead we could use the `set_config` method and avoid having this error get raised altogether

Reply:
It will still run into an error, since we have a version of it validating set_config as well:

https://github.com/sdv-dev/RDT/blob/1c2c8211fd5697e4945bbdd28f2cccb353735455/rdt/hyper_transformer.py#L240-L244

Reply:
When we set the config can't we force the sdtype to match?